src/test-apps/TestDataManagement.cpp

```diff
@@ -327,11 +327,13 @@ class DMTestClient :
         return err;
     }
 
-    void IncompleteIndication(const uint64_t &aPeerNodeId, StatusReport &aReport)
+    virtual void IncompleteIndication(const uint64_t &aPeerNodeId, StatusReport &aReport)
```

Comment:
Unnecessary virtual, but perhaps  a beneficial override?

Reply:
Agreed. See above.

Reply:
As with MockDMPublisher I think this is an error, as nothing calls `DMTestClient::IncompleteIndication(const uint64_t &aPeerNodeId, StatusReport &aReport)`.

Reply:
See https://github.com/openweave/openweave-core/pull/70#discussion_r228686517.